docs/docs/examples/database/AMAZON_RDS.md

```diff
@@ -7,10 +7,16 @@ This example provides instructions for creating an Amazon [RDS DB instance](http
 ## Database creation
 There are two steps for creating the database:
 
+1. [Initialize database server](#1-initialize-database-server)
+2. [Initialize database & user](#2-create-database--user)
+
 ### 1. Initialize database server
 For details on standing up an RDS DB server follow the guide: [Creating an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html){.external}. 
-### 2. Create database
-For details on creating the database user and database itself follow the appropriate guide below:
+### 2. Initialize database & user
```

Comment:
Add empty line above

Reply:
Doing this actually makes no difference to the rendered doc.